src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -79,6 +92,36 @@ export default function LatestProgressUpdateCardPresentation({
                             </span>
                         </span>
                     </Tooltip>
+                    {isDevMode &&
+                        (username === '' ? (
+                            <UserAvatar
+                                userProfileImageUrl={userProfileImageUrl}
+                            />
+                        ) : (
+                            <Tooltip
+                                content={username}
+                                tooltipPosition={{
+                                    top: '-2.5rem',
+                                    right: '-3.9rem',
+                                    width: '10rem',
+                                }}
+                            >
+                                <a
+                                    href={`${USER_MANAGEMENT_URL}?username=${username}`}
```

Comment:
can use Link from next.js instead of hyperlink ?

Reply:
I have discussed with Vinit about it if you check some resolve conversation. Link don't give any additional benefit in this case as we are redirecting user to another website. `Nextjs` preferred Link only for internal routing

Reply:
okay, then resolving this.